core/vm/eips.go

```diff
@@ -303,3 +304,20 @@ func enable6780(jt *JumpTable) {
 		maxStack:    maxStack(1, 0),
 	}
 }
+
+func enable4762(jt *JumpTable) {
+	jt[SSTORE].constantGas = 0
+	jt[SSTORE].dynamicGas = gasSStore4762
+	jt[SLOAD].constantGas = 0
+	jt[SLOAD].dynamicGas = gasSLoad4762
+	jt[BALANCE].dynamicGas = gasBalance4762
+	jt[EXTCODESIZE].dynamicGas = gasExtCodeSize4762
+	jt[EXTCODEHASH].dynamicGas = gasExtCodeHash4762
+	jt[SELFDESTRUCT].dynamicGas = gasSelfdestructEIP4762
+	jt[CREATE].constantGas = params.CreateNGasEip4762
+	jt[CREATE2].constantGas = params.CreateNGasEip4762
```

Comment:
The constant cost for gas is reduced to 1000 according to the eip. But the computation for the total cost depends on the previous way of computing it (eip 3860 if memory serves).